cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -220,6 +220,7 @@ profiles:
 	if err := ioutil.WriteFile(multiProfilesConfig, []byte(fmt.Sprintf(`
 apiVersion: kubescheduler.config.k8s.io/v1beta1
 kind: KubeSchedulerConfiguration
+parallelism: 16
```

Comment:
remove this line. This test is only for things exposed through command line flags.